test/scripts/create-cluster.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This shell script is used to build a cluster and create a namespace.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CLUSTER_NAME="${CLUSTER_NAME}"
+ZONE="${GCP_ZONE}"
+PROJECT="${GCP_PROJECT}"
+NAMESPACE="${DEPLOY_NAMESPACE}"
+
+echo "Activating service-account ..."
+gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
+
+echo "Creating cluster ${CLUSTER_NAME} ... "
+gcloud --project ${PROJECT} beta container clusters create ${CLUSTER_NAME} \
+    --addons=HorizontalPodAutoscaling,HttpLoadBalancing,Istio \
```

Comment:
Do we want to install the knative addon as well?

Reply:
Not sure the knative is addon for gcloud, from official guide, install knative google cloud as this way, but I will have a try later.

Reply:
https://cloud.google.com/run/docs/quickstarts/prebuilt-deploy-gke#create_cluster

Reply:
Seems the knative is not addon currently, and istio in addons cannot work with KFServing, see below, I gave up the addon way... :-)